insights/parsers/mysqladmin.py

```diff
@@ -15,6 +15,38 @@
 from insights.specs import Specs
 
 
+@parser(Specs.mysqladmin_status)
+class MysqladminStatus(LegacyItemAccess, CommandParser):
+    """
+    Module for parsing the output of the ``mysqladmin status`` command.
+    Typical output looks like::
+        Uptime: 1103965 Threads: 1820 Questions: 44778091 Slow queries: 0 Opens: 1919 Flush tables: 1 Open tables: 592 Queries per second avg: 40.561
+    Example:
+        >>> "Uptime" in mysqlstat
+        True
+        >>> mysqlstat['Threads']
+        '1820'
+        >>> mysqlstat['Flush_tables'] == '1'
+        True
+    """
+    def parse_content(self, content):
+        if not content:
+            raise ParseException("Input content is empty.")
+        self.data = {}
+
+        if len(content) == 1:
+            st = list(content)[0].strip()
+            line = st.replace(': ', '=')
+            line_ = line.replace("Slow queries", "Slow_queries").replace("Flush tables", "Flush_tables").replace("Open tables", "Open_tables").replace("Queries per second avg", "Queries_per_second_avg")
+            if line_.startswith("Uptime="):
+                for item in line_.split(None)[0:]:
+                    try:
+                        k, v = item.split('=')
+                        self.data[k.strip()] = v.strip()
+                    except ValueError:
+                        continue
```

Comment:
Is `ValueError` the only exception that gets raised here?  Is it OK to continue if you get an exception?